client/blocks/express-checkout/hooks.js

```diff
@@ -82,6 +84,20 @@ export const useExpressCheckout = ( {
 
 			// Click event from WC Blocks.
 			onClick();
+
+			if ( getExpressCheckoutData( 'taxes_based_on_billing' ) ) {
+				displayExpressCheckoutNotice(
+					__(
+						'Final taxes charged can differ based on your actual billing address when using Express Checkout buttons (Link, Google Pay or Apple Pay).',
+						'woocommerce-gateway-stripe'
+					),
+					'info',
+					[ 'ece-taxes-info' ]
+				);
+				// Wait for the notice to be displayed before proceeding.
+				await new Promise( ( resolve ) => setTimeout( resolve, 700 ) );
```

Comment:
Curious: why `700`? Should we add as comment here if there's a min/max/recommended value for the delay?